demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -997,6 +1003,88 @@ def get_file_description(path, file_type) -> str:
 
 
 def update_api_modules_dependents_rn(
+    pre_release: bool,
+    update_type: Union[str, None],
+    added: Union[list, set],
+    modified: Union[list, set],
+    text: str = "",
+) -> set:
+    """Updates release notes for any pack that depends on API module that has changed.
+    :param
+        pre_release: The file type
+        update_type: The update type
+        added: The added files
+        modified: The modified files
+        id_set_path: The id set path
+        text: Text to add to the release notes files
+
+    :rtype: ``set``
+    :return
+    A set of updated packs
+    """
+    total_updated_packs: set = set()
+    api_module_set = get_api_module_ids(added)
+    api_module_set = api_module_set.union(get_api_module_ids(modified))
+    logger.info(
+        f"[yellow]Changes were found in the following APIModules: {api_module_set}, updating all dependent "
+        f"integrations.[/yellow]"
+    )
+    integrations = get_api_module_from_graph(api_module_set)
+    for integration in integrations:
+        integration_pack = get_pack_name(integration)
+        integration_path = integration
+        integration_pack_path = pack_name_to_path(integration_pack)
+        update_pack_rn = UpdateRN(
+            pack_path=integration_pack_path,
+            update_type=update_type,
+            modified_files_in_pack={integration_path},
+            pre_release=pre_release,
+            added_files=set(),
+            pack=integration_pack,
+            text=text,
+        )
+        updated = update_pack_rn.execute_update()
+        if updated:
+            total_updated_packs.add(integration_pack)
+    return total_updated_packs
+
+
+def get_api_module_from_graph(changed_api_modules):
+    if changed_api_modules:
+        dependent_items = []
+        with Neo4jContentGraphInterface() as graph:
+            logger.info("Updating graph...")
+            update_content_graph(graph, use_git=True, dependencies=True)
```

Comment:
Instead of calling update directly, you can use the `should_update` argument.
```suggestion
        with Neo4jContentGraphInterface(should_update=True) as graph:
```

Reply:
thanks